sfge/src/main/java/com/salesforce/rules/fls/apex/operations/FlsValidationCentral.java

```diff
@@ -142,10 +142,6 @@ public void createExpectedValidations(DmlStatementVertex vertex, SymbolProvider
         }
 
         final List<BaseSFVertex> dmlStatementVertexChildren = vertex.getChildren();
-        if (dmlStatementVertexChildren.size() != validationType.parameterCount) {
-            throw new UnexpectedException(
-                    "Unexpected count of parameters: " + dmlStatementVertexChildren.size());
-        }
```

Comment:
This check was me being extra cautious when I didn't understand ASTs better. This doesn't make sense anymore since ASTs will be standard based on compilation.